src/content/kubernetes/best-practices/index.md

```diff
@@ -35,25 +41,25 @@ You don't want your development environment to be down, because of a minor node
 
 For production usage the cluster should have at least 5 nodes and a buffer of 2 nodes.
 
-#### Master nodes
+#### Control plane nodes
 
-Master nodes matter much more than worker nodes in order to achieve better availability and performance. Several worker nodes can be replaced at the same time with no downtime but the failure of a master node could cause a major issue if it is your only master node. High availavility can be used to reduce downtime to a minimum but sill need to be sized correctly to avoid performance issues.
+Control plane nodes matter much more than worker nodes in order to achieve better availability and performance. Several worker nodes can be replaced at the same time with no downtime but the failure of a control plane node could cause a major issue if it is your only control plane node. [High-availability control planes]({{< relref "/advanced/high-availability/control-plane/index.md" >}}) can be used on AWS to reduce downtime to a minimum but still need to be sized correctly to avoid performance issues.
```

Comment:
```suggestion
Control plane nodes are more critical than worker nodes in achieving better availability and performance. Several worker nodes can be replaced at the same time with no downtime but the failure of a control plane node could cause a major issue if it is your only control plane node. [High-availability control planes]({{< relref "/advanced/high-availability/control-plane/index.md" >}}) can be used on AWS to reduce downtime to a minimum but still need to be sized correctly to avoid performance issues.
```